visualization/app/codeCharta/ui/scenarioDropDown/scenarioDropDown.component.html

```diff
@@ -1,16 +1,33 @@
-<md-menu>
-	<md-button class="scenario-button" ng-click="$mdMenu.open($event)">
+<md-menu class="scenario-drop-down">
+	<md-button class="scenario-button" ng-style="{'color': $ctrl.getButtonColor()}" ng-click="$ctrl.loadScenarios(); $mdMenu.open($event)">
 		<i class="fa fa-star"></i>
 	</md-button>
 	<md-menu-content width="2">
-		<md-menu-item style="min-height: 20px; height: 20px;">
-			<md-button style="min-height: 20px; height: 20px; line-height: 20px; font-size: 9pt; text-align: center;" ng-disabled="true"
-				>Metric Scenario</md-button
-			>
+		<md-menu-item class="scenario-title">
+			<md-button class="title-button" ng-disabled="true">Metric Scenario</md-button>
 		</md-menu-item>
+
 		<md-divider></md-divider>
-		<md-menu-item ng-repeat="scenario in $ctrl._viewModel.scenarios">
-			<md-button ng-click="$ctrl.applyScenario(scenario.name)">{{::scenario.name}}</md-button>
+
+		<md-menu-item class="custom-item" ng-hide="!$ctrl._viewModel.isScenarioChanged">
+			<md-button class="scenario-items" ng-click="$ctrl.showAddScenarioSettings()">Add Custom Scenario</md-button>
+			<md-button ng-click="$ctrl.showAddScenarioSettings()" class="add-scenario-button"
+				><i class="fa fa-plus" aria-hidden="true"></i
+			></md-button>
+		</md-menu-item>
+
+		<md-divider></md-divider>
+
+		<md-menu-item class="scenarios" ng-repeat="scenario in $ctrl._viewModel.scenarios track by $index">
+			<md-button class="scenario-items" ng-click="$ctrl.applyScenario(scenario.name)">{{ scenario.name }}</md-button>
+			<i class="fa fa-video-camera" ng-style="{'color' : $ctrl.getVisibility('view', scenario.name)}"></i>
+			<i class="fa fa-arrows-alt" ng-style="{'color' : $ctrl.getVisibility('area', scenario.name)}"></i>
+			<i class="fa fa-paint-brush" ng-style="{'color' : $ctrl.getVisibility('color', scenario.name)}"></i>
+			<i class="fa fa-arrows-v" ng-style="{'color' : $ctrl.getVisibility('height', scenario.name)}"></i>
+			<i class="fa fa-exchange" ng-style="{'color' : $ctrl.getVisibility('edges', scenario.name)}"></i>
+			<md-button ng-click="$ctrl.removeScenario(scenario.name)" class="remove-scenario-button">
+				<i class="fa fa-minus"></i>
+			</md-button>
```

Comment:
Why don't we remove our `small-action-button` as in the blacklist panels? I think we would have quite less sass for styling the buttons.

Reply:
Turns out its not so easy to use custom styling inside the `md-menu` angular component